app/services/file_path_sanitizer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class FilePathSanitizer
+  def sanitize(file_path, on_error: nil)
+    pathname = Pathname.new(file_path)
+    return pathname.realpath if  pathname.file?
```

Comment:
I'm surprised Rubocop didn't complain about the extra space here.
```suggestion
    return pathname.realpath if pathname.file?
```